backend/app/repositories/base.py

```
@@ -0,0 +1,290 @@
+            # Add updated timestamp
+            updates['updated_at'] = datetime.utcnow()
+
+            stmt = update(self.model).where(
+                and_(*[getattr(self.model, k) == v for k, v in filters.items()])
+            ).values(**updates)
+
+            result = await self.session.execute(stmt)
+            await self.session.commit()
+
+            logger.info(f"Bulk updated {result.rowcount} {self.model.__name__} records")
+            return result.rowcount
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)